webui/src/pages/auth/groups/index.tsx

```diff
@@ -103,13 +107,21 @@ const GroupsContainer = () => {
 
     const router = useRouter();
     const after = (router.query.after) ? router.query.after : "";
+    const prefix = (router.query.prefix) ? router.query.prefix : "";
```

Comment:
Nit!
Since you use `prefix` before `after` (🙃),
Better to place it here before `after`.